wormhole-connect/src/views/v2/Redeem/index.tsx

```diff
@@ -246,23 +255,20 @@ const Redeem = () => {
       <>
         <Box sx={{ position: 'relative', display: 'inline-flex' }}>
           {isTxComplete ? (
-            <TxCompleteIcon
+            <TxCompleteIcon className={classes.txStatusIcon} />
+          ) : isTxRefunded ? (
+            <TxRefundedIcon
+              className={classes.txStatusIcon}
               sx={{
-                width: 105,
-                height: 105,
-                marginTop: 1,
-                marginBottom: 2,
+                color: '#f3bb2b',
```

Comment:
Just an FYI. I'll need to revisit this color code after merging the theming changes. We need one from the new theme palette here.

Reply:
Used theme.pallete instead.